### ISSUE TITLE
Disable logging

### ISSUE THREAD

**Issue by jmikrut** [a7ul/esbuild-node-tsc#56]
Hey there! Great package. I want to use this for all of [Payload](https://payloadcms.com)'s `create-payload-app` invocations, but to do so I'd really like to have a way to disable logs like:

- `Using default config`
- `Built in: XX.YYYms`

Just looking to clean up the terminal output a bit because this doesn't really need to be visible to `create-payload-app` users.

Any thoughts? Thanks for the great package!

**Comment by a7ul** [comment]
Makes sense. 
Could you raise a pr ? 
Maybe an env variable that can disable the output or a `--silent` flag ?

**Comment by jmikrut** [comment]
Yes, someone on my team can get to this for sure. I will put it on our radar 👍
